src/text-overflow/TextOverflow.js

```diff
@@ -51,9 +51,15 @@ export default class TextOverflow extends React.Component {
 
   verifyOverflow = (_state) => {
     const state = _state || this.state;
-    if (state.isOverflowing === false) {
+    if (state.isOverflowing === false && window) {
```

Comment:
`typeof window !== 'undefined'` is nicer, this would be a reference error in `node`